parlai/utils/bpe.py

```diff
@@ -219,6 +248,18 @@ def sync_with_dict(self, dict_agent):
             agent with which we are syncing the dictionary
         """
 
+    def add_special_tokens(self, dict_agent, special_tokens: List[str]):
+        """
+        Add special tokens to the tokenizer.
+
+        These tokens are treated as discreet tokens, and prioritized over the BPE
+        tokenization.
+        """
+        # note, HF ByteLevelBPE tokenizer handles special tokens itself in
```

Comment:
Hmm what is the special way - is it something we should be concerned about if using this?

Reply:
No, I just wanted to point out this is all no-ops if using bytelevelbpe.